src/System.IO.FileSystem/src/System.IO.FileSystem.builds

```diff
@@ -10,22 +10,13 @@
     </Project>
     <Project Include="System.IO.FileSystem.csproj">
```

Comment:
Since it's a split package I agree.  The only folks that'd get an update this way are folks building standalone which is the minority.  If our customers wanted a fix for this they'd want it in the shared framework which means servicing.